frontend/src/components/VideoCard/VideoCard.js

```diff
@@ -8,39 +8,52 @@ import {
   Grid,
 } from '@material-ui/core';
 import CheckCircleIcon from '@material-ui/icons/CheckCircle';
-import { missingImageIcon } from '../../assets';
 import './styles.css';
+import { missingImageIcon } from '../../assets';
+import { getVideoThumbnail } from '../../api/VideoAPI';
 
-const VideoCard = ({ title, thumbnail, onSelect, id, isSelected }) => (
-  <Card
-    className="card"
-    variant="outlined"
-    style={{ border: `${isSelected ? '0.5px solid blue' : ''}` }}
-  >
-    <CardHeader
-      classes={{
-        content: 'card_content',
-      }}
-      disableTypography
-      title={
-        <Typography noWrap gutterBottom variant="h6">
-          {title}
-        </Typography>
-      }
-      action={
-        <Grid item>
-          <IconButton onClick={() => onSelect(id)}>
-            <CheckCircleIcon color={isSelected ? 'primary' : 'inherit'} />
-          </IconButton>
-        </Grid>
-      }
-    />
-    <CardMedia
-      component="img"
-      className="card_image"
-      src={thumbnail || missingImageIcon}
-    />
-  </Card>
-);
+const VideoCard = ({ title, onSelect, id, isSelected }) => {
+  const [thumbnail, setThumbnail] = useState(undefined);
+  useEffect(() => {
+    getVideoThumbnail(id)
+      .then((response) => setThumbnail(URL.createObjectURL(response.data)))
+      .catch(() => setThumbnail(missingImageIcon));
+  }, []);
+  const handleClick = () => {
+    window.location.href = `/player/${id}`;
+  };
```

Comment:
add some empty lines between each const/useEffect, now it seems like everything is squashed 😄

Reply:
Did.. Did you just.. DID YOU JUST ASKDMEETOPUTSOMEMPTILINES?!?!?!?!